communityrust.properties

```diff
@@ -1,14 +1,21 @@
 category=Languages
 description=Code Analyzer for Rust
 homepageUrl=https://github.com/elegoff/sonar-rust
-archivedVersions=0.0.6
-publicVersions=0.0.7
+archivedVersions=0.0.6,0.0.7
+publicVersions=0.0.8
 
 defaults.mavenGroupId=org.elegoff
 defaults.mavenArtifactId=community-rust-plugin
 
+0.0.8.description=Unit test detection 
+0.0.8.sqVersions=[8.9,LATEST]
+0.0.8.date=2022-01-28
+0.0.8.changelogUrl=https://github.com/elegoff/sonar-rust/releases/tag/v0.0.8
+0.0.8.downloadUrl=https://github.com/elegoff/sonar-rust/releases/download/v0.0.8/community-rust-plugin-0.0.8.jar
+
+
 0.0.7.description=Update with latest Rust grammer and refeshed list of Clippy list 
-0.0.7.sqVersions=[8.9,LATEST]
+0.0.7.sqVersions=[8.9,9.2.*]
```

Comment:
I know 9.3 wasn't out when you submitted this, but it is by the time I saw it. :(
Please update the end of this range